Formula/aws-console.rb

```diff
@@ -0,0 +1,19 @@
+class AwsConsole < Formula
+  desc "Command-line to use AWS CLI credentials to launch the AWS console in a browser"
+  homepage "https://github.com/aws-cloudformation/rain"
+  url "https://github.com/aws-cloudformation/rain/archive/v1.1.1.tar.gz"
+  sha256 "5faf25a87a6fc0fde19edcf2fe6f26c81990da91d13fbf3858b1eb33711b0ebd"
+  license "Apache-2.0"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args, "cmd/aws-console/main.go"
+  end
+
+  test do
+    # No other operation is possible without valid AWS credentials configured
+    _, stderr, = Open3.capture3("#{bin}/aws-console")
+    assert_match("could not establish AWS credentials; please run 'aws configure' or choose a profile", stderr)
```

Comment:
```suggestion
    assert_match "could not establish AWS credentials; please run 'aws configure' or choose a profile", shell_output("#{bin}/aws-console")
```

Reply:
This suggestion won't work as I'm capturing the failure message printed to `stderr`. `shell_output` only appears to capture `stdout`

Reply:
Add a bash redirect on the end. It's what all other formulae with this issue do.

Reply:
I did try with `"#{bin}/aws-console 2>&1"` but the test still fails with `expected to be =~ <"">`